### PR TITLE
added a basic docker-compose file

### PR DESCRIPTION
**Pull request by Kk-ships**

### CODE REVIEW

**Review by nathanjisaac — COMMENTED** [review]
Is `/data` the correct volume location for the actual server data?

**Comment by matthewbahr** [comment]
Isn't it better to put the data in /usr/share/?

That's what my homeassistant docker compose does.

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+services:
+  actual_server:
+    container_name: actual_server
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    ports:
+      - "5006:5006"
+    volumes:
+      - ./data:/data
```
Comment (nathanjisaac) [inline docker-compose.yml:11]:
```suggestion
      - ./server-files:/app/server-files
      - ./user-files:/app/user-files
```

I think this is what you want for the volumes. 

I'm matches these in the `Dockerfile`. 

```Dockerfile
RUN mkdir ./server-files
RUN mkdir ./user-files
```

**Comment by nathanjisaac** [comment]
I just tested this locally on my machine. Seems to work good.

**Review by fedme — APPROVED** [review]
Looks good to me! (I am not a maintainer, just a random dev :smiley:)

**Comment by jlongster** [comment]
Is this still up-to-date given the latest docker file changes?

**Comment by Kk-ships** [comment]
@jlongster docker-compose file would be Dockerfile agnostic. I just builds a docker container based on Dockerfile contents.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+version: "3"
+services:
+  actual_server:
+    container_name: actual_server
+    build:
+      context: ./
+      dockerfile: Dockerfile
```
Comment (nathanjisaac) [inline docker-compose.yml:7]: @jlongster It's dynamically referenced here.

**Comment by jlongster** [comment]
Yes but now we have two different docker files after #22, `Dockerfile` and `Dockerfile.alpine`. Does that make a difference?

**Comment by jlongster** [comment]
Thanks, I'll add  this!

**Pull request merged**